sdk/identity/azure-identity/src/main/java/com/azure/identity/ManagedIdentityCredential.java

```diff
@@ -55,9 +57,10 @@
             }
         } else if (configuration.contains(Configuration.PROPERTY_AZURE_CLIENT_ID)
```

Comment:
Would this be the value of AZURE_CLIENT_ID only, or does it also capture the constructor option?

Reply:
Asking because token exchange doesn't require a value for AZURE_CLIENT_ID. If that variable has no value, we require one from the user via a constructor parameter or option bag, however ManagedIdentityCredential takes it.

Reply:
yeah, clientId value is accepted.